src/sync/incomingSyncUtils.js

```diff
@@ -372,13 +372,18 @@ export function createOrUpdateRecord(database, settings, recordType, record) {
       const item = getObject(database, 'Item', record.item_ID);
       itemBatch.item = item;
       item.addBatchIfUnique(itemBatch);
-      const packSize = parseNumber(record.pack_size);
+      let incomingPackSize = parseNumber(record.pack_size);
+      let packSize = 1;
+      if (transaction.isExternalSI) {
```

Comment:
As per above packSize === 1 comment, it's for external supplier invoices to keep transaction batches as is. Since they can have pack size other then 1, and we want to keep the entered pack size